include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -180,7 +182,7 @@ class logging_resource_adaptor final : public device_memory_resource {
     msg += ",";
     msg += std::to_string(bytes);
     msg += ",";
-    msg += std::to_string(reinterpret_cast<uintptr_t>(stream));
+    msg += std::to_string(reinterpret_cast<uintptr_t>(cudaStream_t{stream}));
```

Comment:
`rmm::stream_t` could provide a `to_string()` member to simplify this...

Reply:
I added an explicit conversion operator to `uintptr_t` to make this more concise. Is that an ok compromise?